lib/product-import.coffee

```diff
@@ -47,7 +50,7 @@ class ProductImport
 
   _resetCache: ->
     @_cache =
-      productType: {}
+      productType: {} # productTypeId : productType map
```

Comment:
because it also stores id to sameForAllAttributes in the product type... so thought of using a generic name.